src/cpp/src/scheduler.hpp

```diff
@@ -242,7 +242,10 @@ class Scheduler {
                     // add information to scheduler_output
                     {
                         scheduler_output.m_scheduled_sequence_groups_ids.push_back(sequence_group_id);
-                        scheduler_output.m_block_tables[seq_id] = m_block_manager.get_block_tables(seq_id);
+                        for (auto& sequence : sequence_group->get_running_sequences()) {
+                            uint64_t seq_id = sequence->get_id();
+                            scheduler_output.m_block_tables[seq_id] = m_block_manager.get_block_tables(seq_id);
+                        }
```

Comment:
when we are in prompt phase, we don't have multiple sequences. Is it a new case?

Reply:
prompt phase does not support multiseq mode. But we need it to start main model with candidates on first iteration

Reply:
Implemented `first inference` to avoid of several prompt scheduling